ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -21,12 +23,21 @@ import {CronWorkflowEditor} from '../cron-workflow-editor';
 import '../../../workflows/components/workflow-details/workflow-details.scss';
 import './cron-workflow-details.scss';
 
+const storage = new ScopedLocalStorage('ListOptions');
+
 export function CronWorkflowDetails({match, location, history}: RouteComponentProps<any>) {
     // boiler-plate
     const {navigation, notifications, popup} = useContext(Context);
     const queryParams = new URLSearchParams(location.search);
 
     const [namespace] = useState(match.params.namespace);
+    const [pagination] = useState<Pagination>(() => {
+        const savedPaginationLimit = storage.getItem('options', {}).paginationLimit || undefined;
+        return {
+            offset: queryParams.get('offset') || undefined,
+            limit: parseLimit(queryParams.get('limit')) || savedPaginationLimit || 50
+        };
+    });
```

Comment:
I see that you copied this from [the code](https://github.com/argoproj/argo-workflows/blob/986b069f34ed5336b6f754cfc6709c30b302c768/ui/src/app/workflows/components/workflows-list/workflows-list.tsx#L56) in the Workflows List, however this does not make sense here.
This is currently creating `state` that is not used -- meaning it isn't stateful to begin with.

I notice that `namespace` and `name` in this function aren't stateful either, those should be converted to plain `const`s as well.

More specifically, this code allows the initial pagination to be set via `localStorage` and query params, but there is no way to change it after. In the Workflows List page, there is the `PaginationPanel` that can change those.

We also don't want it to be changeable -- it should be a hard-coded `50`

```suggestion
```

Reply:
I see, thanks.
Now I understand why you did not add the pagination feature to this page.
I may issue an additional PR in the future when I get a chance, but for now, please let this be the response.

Reply:
A PR to add a link at the bottom would be good!